File: java/src/main/java/com/iteso/nintendo/attacks/Psybeam.java
package com.iteso.nintendo.attacks;

public class Psybeam implements Attack {
    static int damage;
    private static final String name = "Psybeam";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/Psybeam.py
class Psybeam:
    def __init__(self):
        self.damage = None
        self.name = 'Psybeam'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
